resources/helm/v1.0/istio/charts/grafana/values.yaml

```diff
@@ -0,0 +1,93 @@
+#
+# addon grafana configuration
+#
+enabled: false
+replicaCount: 1
+image:
+  repository: grafana/grafana
```

Comment:
This should have been changed by the grafana patch scripts. Did the scripts run against this?

Reply:
They should have been.  These are pulled in from the maistra-1.0 branch and should match what we're using in 1.0.4.  I'll have to verify, but this might be overridden in the templates (profiles).

Reply:
Verified.  This isn't use and is overridden by a string value in the templates, e.g. https://github.com/Maistra/istio-operator/blob/maistra-1.1/deploy/smcp-templates/servicemesh#L16